README.md

```diff
@@ -129,67 +132,67 @@ public class DefaultController : Controller
+If you need to render BEM-block from the Razor template you have to use the `@Html.Bem` helper and provide the essential bemjson for it.
 ```cs
 @Html.Bem(new { block = "my-block", data = Model })
 ```
 
-В разделе `bemSettings` файла Web.config вы можете настраивать, каким способом будут выбираться бандлы с bemhtml шаблонами для http-запросов. Возможны 3 варианта мэппинга запросов на бандлы:
+In the `bemSettings` section of Web.config file you can setup how to selected the template-bundles for http-requests. There are 3 choices of request-to-bundle mapping available:
 
-1. Один общий бандл на всё приложение - его название можно задать в параметре `DefaultBundle` (по умолчанию `default`):
+1. The single bundle per application - it's name can be set by `DefaultBundle` parameter (`default` by default):
 
   ```xml
   <bemSettings Mapper="Single" DefaultBundle="index" />
   ```
-1. Отдельный бандл для каждого серверного контроллера:
+1. Separate bundle for each server-side controller:
 
   ```xml
   <bemSettings Mapper="ByController" />
   ```
-  Название бандла определяется по названию контроллера: слова разделяются дефисами, приводятся к нижнему регистру, удаляется суффикс "controller" и добавляется префикс `p-` (например, `MainPageController` → `p-main-page`).
-1. Собственный мэппер - есть возможность написать свой класс мэппера и указать его название в параметре `Mapper`:
+  The bundle name is formed by controller name: all words are separated by dashes, all letters changed to lower case, the "controller" suffix is deleted and `p-` prefix is added (for example, `MainPageController` → `p-main-page`).
+1. The custom mapper - you can write your own mapper class and set it's name in `Mapper` parameter:
 
   ```xml
   <bemSettings Mapper="MyApplication.MyNamespace.InnerNamespace.MyMapperClass" />
   ```
-Класс мэппера должен быть унаследован от базового класса [System.Web.Bem.BundleMappers.Mapper](System.Web.Bem/BundleMappers/Mapper.cs) и реализовывать метод `abstract string GetBundleName(ControllerContext context)`, получающий на вход контекст запроса и возвращающий название бандла. Также, при желании, можно переопределить метод `virtual string GetBundlePath(string bundleName)`, возвращающий по названию бандла путь к файлу с bemhtml шаблонами (по умолчанию формируется путь `<RootDir>\<bundleName>\<bundleName>.bemhtml.js`)
+Your mapper class must be inherited from the base class named [System.Web.Bem.BundleMappers.Mapper](System.Web.Bem/BundleMappers/Mapper.cs) and it must implement the `abstract string GetBundleName(ControllerContext context)` method, which receives a request context as an input parameter and returns the bundle name for this request. Also you can redefine the `virtual string GetBundlePath(string bundleName)` method, which returns the bemhtml-bundle full path by the bundle name (by default the path `<RootDir>\<bundleName>\<bundleName>.bemhtml.js` is formed).
```

Comment:
which returns the bemhtml-bundle full path by the bundle name -> which returns the full path to the bemhtml bundles for the bundle name